qa/workunits/fs/test-volumes.sh

```diff
@@ -22,10 +22,79 @@ ceph fs volume create bar
 ceph fs volume ls | grep bar
 ceph fs volume rm bar
 
+### basic subvolume tests
+
 # subvolumes on $EXISTING
 ceph fs subvolume create $EXISTING sub1
 ceph fs subvolume create $EXISTING sub2
+# make sure it exists
+ceph fs subvolume getpath $EXISTING sub1
+ceph fs subvolume getpath $EXISTING sub2
+
+# creating an already existing subvolume is idempotent
+ceph fs subvolume create $EXISTING sub1
+
+# remove subvolumes and verify
+ceph fs subvolume rm $EXISTING sub1
 ceph fs subvolume rm $EXISTING sub2
+# make sure its removed. note that this is a negative test
+ceph fs subvolume getpath $EXISTING sub1 && exit 1
+ceph fs subvolume getpath $EXISTING sub2 && exit 1
+
+# deleting an already deleted subvolume is idempotent
+ceph fs subvolume rm $EXISTING sub1
+
+# create and delete subvolumes w/ same subvol name as above
+# this would test subvole purge queue when that's implemented
+# (currently, deleting subvolumes is an synchronous operation)
```

Comment:
Don't follow how this would test asychronous purge. Can you please explain?

Reply:
this does not test async purge as such, but the side effect of recreating a subvolume with the same name in case it returns -EEXIST. I think we can ignore this for now...